controllers/temprole_controller.py

```diff
@@ -123,6 +123,28 @@ async def view_temproles(user: User, interaction: Interaction):
 
         await interaction.response.send_message(response, ephemeral=True)
 
+    @staticmethod
+    async def view_users(role: Role, interaction: Interaction):
+        temproles = DB().get_temprole_users(role.id, interaction.guild.id)
```

Comment:
This variable name really isn't descriptive, consider `users_with_temprole` or similar